certbot/tests/reverter_test.py

```diff
@@ -347,7 +347,7 @@ def test_finalize_checkpoint_cannot_title(self, mock_move):
         self.assertRaises(
             errors.ReverterError, self.reverter.finalize_checkpoint, "Title")
 
-    @mock.patch("certbot.reverter.misc.os_rename")
+    @mock.patch("certbot.reverter.filesystem.replace")
     def test_finalize_checkpoint_no_rename_directory(self, mock_rename):
```

Comment:
nit: I think we should change this variable name to `mock_replace`.

Reply:
Done.